src/openforms/emails/templates/emails/wrapper.html

```diff
@@ -10,7 +10,9 @@
                     <td align="left" style="padding: 20px; color: {{ style.header.fg }}; background-color: {{ style.header.bg }};">
                         {% if logo_url %}
                             <a style="text-decoration: none;" href="{{ main_website_url }}">
-                                <img style="{% spaceless %}
+                                <img {% if style.logo.width %}width="{{ style.logo.width_attr }}" {% endif %}
+				     {% if style.logo.height %}height="{{ style.logo.height_attr }}" {% endif %}
+				    style="{% spaceless %}
```

Comment:
if it's set to `auto`, this will still output the `width`/`height` attribute but with an empty value. Does that make sense? What happens in e-mail clients when that occurs?

Reply:
Only the value for the img tag attribute is empty; the CSS values are untouched. Hence it should display correctly (I've confirmed this for web clients; still waiting to hear from Joeri for MS Outlook's desktop client, which is the real culprit).

Reply:
My bad, I forgot to modify the `if`-test in the template to check the `attr` values of the `img` attributes. Now the `width` will simply not be present if it's set to `auto`

Reply:
please add integration tests for the actual e-mail rendering in that case - the template filter unit tests pass but didn't catch this problem, this implies you're lacking some tests :)